src/impl_raw_views.rs

```diff
@@ -85,7 +85,7 @@ where
             }
```

Comment:
see assertion `dimension::strides_non_negative(strides).unwrap();` above - that should ensure the scenario you mention can't happen (it's a debug assertion, users are trusted - it's an unsafe function and it's documented strides must be positive)